hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/AWSCredentialProviderList.java

```diff
@@ -100,7 +107,9 @@ public AWSCredentialProviderList() {
    */
   public AWSCredentialProviderList(
```

Comment:
Neither of the old constructors seem to be used. Do we need to keep them?

I think it would be nice to remove all references to v1 from this class, and instead move the adapter call to the "config" code.

Reply:
(maybe a non-goal... I also see e.g. `getCredentials()`)